phpstan-baseline.neon

```diff
@@ -0,0 +1,62 @@
+parameters:
```

Comment:
pslam AND phpstan?

Reply:
Yes sir!

They actually complement each other and they are very easy to run.